website/docusaurus.config.js

```diff
@@ -544,7 +544,7 @@ module.exports = {
                     },
                     {
                         from: '/topics/feature-flags/tutorials/react/implementing-feature-flags',
-                        to: '/feature-flag-tutorials/react/implementing-feature-flags',
+                        to: '/feature-flag-tutorials/react/',
```

Comment:
Did the previous `to` URL exist? If so, we should probably redirect from that one too. Something like this:

```suggestion
                        from: [ '/topics/feature-flags/tutorials/react/implementing-feature-flags', '/feature-flag-tutorials/react/implementing-feature-flags'],
                        to: '/feature-flag-tutorials/react/',
```

Not sure why github is saying everything has changed, though 🤔

Reply:
Good catch. I think the only redirect we need is from 
`/feature-flag-tutorials/react/implementing-feature-flags` to `/feature-flag-tutorials/react`

I don't think `/topics/feature-flags/tutorials/react/implementing-feature-flags` ever existed - no need to redirect from that.

Not sure about the trailing slash (`react` vs `react/`) or not - something to be tested.